exercises/Python_Principles_2.ipynb

```diff
@@ -252,37 +231,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "5\n",
-      "7\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "x = 5 # Global scope\n",
     "def printNumber():\n",
     "    print(x)\n",
     "    y = x + 2 # Local scope\n",
-    "    print(y) \n",
-    "y\n",
-    "printNumber()\n"
+    "    print(y)\n",
+    "\n",
+    "printNumber()"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "In the code snippet above, the first line `x = 5` means that variable is in global scope. On line two to five we define a function `printNumber()` don't worry about what it means at the moment think of it as a way to make local scope when we define variables inside this function. \n",
     "\n",
-    "We print `x` which outputs `5`. On line four we declare the variable `y` with `x + 2`, here `y` has local scope as its inside the function, `y` evaluates to `7` because we can access the variable `x` within the funciton as it has global scope. \n",
-    "\n",
-    "Notice on line six we try to display `y` but the variable `y` is not defined outside of the function. What we mean to say is `y` has local scope within the function and is not available elsewhere. "
+    "We print `x` which outputs `5`. On line four we declare the variable `y` with `x + 2`, here `y` has local scope as its inside the function, `y` evaluates to `7` because we can access the variable `x` within the funciton as it has global scope. "
```

Comment:
Yep cleaner explanation, there's a typo here "funciton", so easy to do! If you want to change it!